### PR TITLE
ci: restore buster folder

### PR DESCRIPTION
**Pull request by morrisonlevi**
### Description

PROF-8286

I accidentally forgot to restore this before merging #2432. Also prepares the bookworm version number for the next build.

### Reviewer checklist
- [ ] Test coverage seems ok.
- [ ] Appropriate labels assigned.

### CODE REVIEW

File: dockerfiles/ci/bookworm/.env [hunk 1]
```diff
@@ -1,2 +1,2 @@
 BOOKWORM_CURRENT_VERSION=1
-BOOKWORM_NEXT_VERSION=1
+BOOKWORM_NEXT_VERSION=2
```
Comment (realFlowControl) [inline dockerfiles/ci/bookworm/.env:2]: Is this change actually related?

Comment (morrisonlevi) [reply dockerfiles/ci/bookworm/.env:2]: It's this part:

> Also prepares the bookworm version number for the next build.

Which is something which also should have been done before that PR was merged.

**Review by realFlowControl — APPROVED** [review]
